arangod/RocksDBEngine/RocksDBEngine.h

```diff
@@ -577,6 +577,23 @@ class RocksDBEngine final : public StorageEngine {
   std::deque<RocksDBKeyBounds> _pendingCompactions;
   /// @brief number of currently running compaction jobs
   size_t _runningCompactions;
+
+  // frequency for throttle in milliseconds
+  uint64_t _throttleFrequency = 60 * 1000; 
```

Comment:
```suggestion
  uint64_t _throttleFrequency = 5 * 1000; 
```

Reply:
I don't object to changing the default value, we should do it in a separate PR after this one.